src/components/internalSettings/AccordionSection.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  Input,
+  InputGroup,
+  Select,
+} from "@chakra-ui/react";
+import { useForm } from "react-hook-form";
+import React from "react";
+import axios from "axios";
+
+const AccordionSection: React.FC = (props: any) => {
```

Comment:
Instead of having the props as any, can you add a `Props` interface and add typings on what this component expects as inputs?